client/plots/sampleScatter.js

```diff
@@ -204,15 +204,17 @@ class Scatter {
 			chartType: 'sampleScatter',
 			settingsKey: 'minDotSize',
 			title: 'Minimum area, in square pixels',
-			min: 0
+			min: 9,
+			max: 3000
 		}
 		const maxDotSizeOption = {
 			label: 'Max area',
 			type: 'number',
 			chartType: 'sampleScatter',
 			settingsKey: 'maxDotSize',
 			title: 'Maximum area, in square pixels',
-			min: 0
+			min: 9,
+			max: 3000
```

Comment:
why is max 3000? I think that's too large. in all-pharma, I can see the dot size increasing and overlapping the scale label.